### ISSUE TITLE
The training accuracy of the custom dataset is close to 0

### ISSUE THREAD

**Issue by pengc-bjtu** [wusize/ovdet#52]
Hello author, I trained the custom dataset with the method you mentioned earlier. At first, the training loss was NAN, and then after I used gradient clipping, the loss was normal, but the accuracy was close to 0. I feel like there's something wrong with the configuration file, can you help me analyze it?my  wechat account is “pctthappy”
